pkg/services/images.go

```
@@ -133,8 +133,17 @@ func (s *ImageService) getImageContentSourcesRepositories(image *models.Image) (
 	// create a slice of repos uuids
 	csReposUUID := make([]string, 0, len(image.ThirdPartyRepositories))
 	for _, repo := range image.ThirdPartyRepositories {
-		csRepo, err := s.Repositories.GetRepositoryByName(repo.Name)
+		csRepo, err := s.Repositories.GetRepositoryByUUID(repo.UUID)
 		if err != nil {
+			if err == repositories.ErrRepositoryNotFound {
+				// only log the error and ignore any repository that does not exist
+				s.log.WithFields(log.Fields{
+					"repository_name": repo.Name,
+					"repository_uuid": repo.UUID,
+					"error":           err.Error(),
+				}).Error("repository was not found on content-sources")
+				continue
```

LGTM, just double checking, if the repo wasn't found
do we want to ignore the error and continue the execution?

---

Exactly that's intentional, eg we ignore the error of the repo was not found on content-sources.